types/hapi__content/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for @hapi/content 6.0.0
+// Project: https://github.com/hapijs/subtext#readme
+// Definitions by: Sebastian Malton <https://github.com/nokel81>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export interface ContentType {
+  mime: string;
+  charset?: string;
```

Comment:
It can be seen from the library tests that this field is optional, https://github.com/hapijs/content/blob/master/test/index.js#L52-L57

Even though it does not appear in the documentation.